dev-manuals/test-plans/lecturer-interface-frontend/plans/u-lecturer-interface-08-switch-to-world-view.md

```diff
@@ -0,0 +1,34 @@
+# Lecturer-Interface: Switch to world view (`u.lecturer-interface-8`)
+
+
+Version: V1.0, 09.08.2022 \
+Author: Max Kästner
+
+## Description
+
+The view can be switched to a world view within a course with the sidebar.
+
+## Precondition
+
+At least one course is created. The tester is on the view of the course (`/courses/<course-id>`)
+
+## Postcondition
+
+The page displays a (selected) world with its dungeons and its configuration possibilities
+
+## Typical procedure
+
+1. Click on a world on the sidebar (worlds are typically displayed with a map icon)
+2. The sidebar expands and the the world row shows its childs which contains its dungeons, npc and minigame views.
+3. The view gets changed and loads the world and its configuration possibilities
+
+## Alternative procedures
+
```

Comment:
Can you cancel that?

Reply:
No? It is just a redirect with a sidebar